### ISSUE TITLE
[Feat]: 

### ISSUE THREAD

**Issue by OliverwengFiltered** [coder/code-server#5826]
Hi there,

can we add a cli option to disable the pop up on the lower right corner when web is served under a particular port ?

Or disable the whole "Ports" feature completely ?

currently the port proxy feature seems has some "relative path"  issue if the code server is already behind a reverse proxy, so better to be disabled to avoid confusion.

Thanks

<img width="1270" alt="Screenshot 2022-11-21 at 11 27 43 AM" src="https://user-images.githubusercontent.com/28814939/203107942-5970f0d1-75fb-4134-bd70-9ff9424ac764.png">

<img width="954" alt="Screenshot 2022-11-21 at 11 28 06 AM" src="https://user-images.githubusercontent.com/28814939/203107938-2a7fb1bc-f202-4ce4-b594-7061aecaeb4b.png">

**Comment by OliverwengFiltered** [comment]
cc @jsjoeio @code-asher

**Comment by jsjoeio** [comment]
Yes! Seems like we'd need to add a CLI flag and then use it here, with the default set to `true` https://github.com/coder/code-server/blob/main/patches/proxy-uri.diff#L178

**Comment by code-asher** [comment]
Does disabling the "auto forward ports" option do anything?

**Comment by code-asher** [comment]
Still might be a good idea to have a CLI flag to disable it completely if needed for security reasons.

**Comment by jsjoeio** [comment]
In your VS Code Settings:

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/205756494-3a0fed42-abba-4333-94ff-537e2bbbd3be.png">

**Comment by OliverwengFiltered** [comment]
@jsjoeio thanks for the direction, I will test it out.

but agreed, a cli option would be nicer to give user ability to turn it off on start

**Comment by OliverwengFiltered** [comment]
> 

confirmed the `auto forward ports` option works.
thanks @code-asher

**Comment by jsjoeio** [comment]
Moving to Backlog for now. If more people want this, we can reconsider a CLI flag.
